src/config/cambodia/layers.json

```diff
@@ -1642,6 +1642,11 @@
   "kh_incident_report": {
     "title": "Disaster impact report",
     "type": "point_data",
+    "admin_level_display": true,
+    "path": "https://prism-api.ovio.org/kobo/forms",
+    "adminCode": "Commune",
```

Comment:
The config should be using snake_case whenever possible, the config loader should translate everything into camelCase automagically :)